### ISSUE TITLE
Heading Structure - only works for the first browser tab

### ISSUE THREAD

**Issue by accessiblewebuk** [ThePacielloGroup/WebAccessibilityToolbar#111]
IE11, Windows 10, WAT 2015-05-20
Steps to reproduce:
1. Open IE11
2. Open a page
3. Right click, duplicate tab (or just open a second tab with any page)
4. Heading structure

It should open a new tab showing the heading structure, but instead it opens a new tab with the same page (as the second tab).

**Comment by patrickhlauke** [comment]
I'm afraid WAT is no longer in active development and no future fixes are currently planned. I'll keep this around until we get a chance to triage for possibly one last major bugfix release before completely closing it.

**Comment by accessiblewebuk** [comment]
Fair enough - this isn't a serious issue anyway, it can be fairly easily avoided.
